corehq/util/tests/test_quickcache.py

```diff
@@ -204,3 +212,59 @@ def by_name(name):
         # values with encodings other than utf-8 still produce different keys
         self.assertEqual(by_name(name_latin1), 'VALUE')
         self.assertEqual(self.consume_buffer(), ['local miss', 'shared miss', 'called'])
+
+    def _test_skippable(self, fn, skip_kwarg):
+        name = 'name'
+        self.assertEqual(fn(name), 'VALUE')
+        self.assertEqual(self.consume_buffer(), ['cache miss', 'called', 'cache set'])
+        self.assertEqual(fn(name), 'VALUE')
+        self.assertEqual(self.consume_buffer(), ['cache hit'])
+
+        self.assertEqual(fn(name, **{skip_kwarg: True}), 'VALUE')
+        self.assertEqual(self.consume_buffer(), ['called', 'cache set'])
+        self.assertEqual(fn(name), 'VALUE')
+        self.assertEqual(self.consume_buffer(), ['cache hit'])
+
+    def test_skippable(self):
+        @quickcache(['name'], cache=_cache_with_set, helper_class=SkippableQuickCache)
+        def by_name(name, skip_cache=False):
+            BUFFER.append('called')
+            return 'VALUE'
+
+        self._test_skippable(by_name, skip_kwarg='skip_cache')
```

Comment:
Worth adding another of these what calls `skippable_quickcache` directly to test that interface? Something like

```
def test_skippable_decorator(self):
    @skippable_quickcache(['name'], cache=_cache_with_set, skip_arg='force')
    def by_name(name, skip_cache=False):
        BUFFER.append('called')
        return 'VALUE'

    self._test_skippable(by_name, skip_kwarg='force')
```